app/templates/components/application/site-header.hbs

```diff
@@ -88,17 +90,10 @@
             {{link-to (t "header.groups") "groups.index" class="nav-link"}}
           </li>
           {{! Feedback}}
-          <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle {{if isFeedbackRoute "active"}}" href="#" data-toggle="dropdown" data-href-to-ignore=true>
+          <li class="nav-item nolt-button">
+            <a class="nav-link" href="#"  data-href-to-ignore=true>
```

Comment:
We should probably link this to `https://feedback.kitsu.io/` to still function in case the Nolt widget breaks or gets blocked